### DISCUSSION TITLE
Direct link to *.bin in docs?

### DISCUSSION THREAD

**Discussion by stnbu** [rustformers/llama-rs#336] [General]
As a total llm newb I had a heck of a time finding e.g. https://huggingface.co/rustformers/redpajama-3b-ggml/blob/main/RedPajama-INCITE-Base-3B-v1-q4_0.bin

There's also a lot of coded info in the file names that intimidate me. A tl;dr on this would allow me to "read" file names which would be a big boost.

I'd be happy to _contribute_ docs, give me an idea of what's welcome/needed.

**Comment by LLukas22** [comment]
That's a reasonable suggestion. At the moment, the naming conventions are auto-generated during the conversion process. If you're interested, you could contribute a brief guide on "Model Downloading 101" to the documentation section of this repository. I could then reference this guide across all our Hugging Face repositories.

Since I'm intimately familiar with the process and the repositories, I might overlook some points that could be confusing for newcomers. You could aim to address anything that posed a challenge to you and provide a concise explanation. If there are any aspects you're unsure about, I could certainly add in more details later.

**Comment by stnbu** [comment]
Excellent! Thank you for the the clarity, etc. I will try to think of what else one might want to know. Maybe "gentle guide to the models universe" or roughly that theme. I need to know myself afterall.

**Reply by LLukas22** [reply]
Just a heads up the format will change again in the near future: https://github.com/ggerganov/ggml/pull/302

**Comment by philpax** [comment]
In addition to what Lukas has mentioned, we have a known good models list: https://github.com/rustformers/llm/blob/main/doc/known-good-models.md (which I've just noticed is not linked obviously from the README. Will fix.)

We're hoping that the GGUF format will help make this much simpler, and you'll be able to download GGUF models and get going quickly. There will still be some work until we get to that stage, but this should be less troublesome soon!
